contrib/automation_tests/orbit_capture_compatibility.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright (c) 2022 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.capture_window import Capture
+from test_cases.connection_window import ConnectToStadiaInstance, FilterAndSelectFirstProcess, LoadCapture
+from test_cases.live_tab import VerifyOneFunctionWasHit
+from test_cases.main_window import DismissDialog, RenameMoveCaptureFile
+from test_cases.symbols_tab import WaitForLoadingSymbolsAndCheckModule
+"""Verify Capture compatibility
+
+This script contains functionality to record a capture with one (new) version of Orbit and then
+test that that this capture can be loaded with another (old) version of Orbit. This is used to test
+that a capture file produces by a (new) Orbit can be read by another (old) version of Orbit.
+
+This needs to be called with one of two values, `record_capture` or `load_capture` and a path to
```

Comment:
Pers Pref: 
This needs to be called with two arguments: 1) one of `record_capture` or `load_capture` , and 
2) a path to where the capture will be saved / from where it will be loaded.